packages/client/hmi-client/src/workflow/ops/model-config/tera-model-config-table.vue

```diff
@@ -0,0 +1,206 @@
+<template>
+	<Datatable
+		:value="data"
+		v-model:expanded-rows="expandedRows"
+		dataKey="id"
+		:row-class="rowClass"
+		class="p-datatable-sm"
+		:class="{ 'hide-header': hideHeader }"
+	>
+		<Column expander style="width: 5rem" />
+		<Column field="id" header="ID" style="width: 10%"></Column>
+		<Column field="name" header="Name" style="width: 15%"></Column>
+		<Column field="type" header="Value type" style="width: 10%">
+			<template #body="slotProps">
+				<Button
+					text
+					v-if="slotProps.data.type === ParamType.MATRIX"
+					icon="pi pi-table"
+					@click="openMatrixModal(tableType, slotProps.data.id)"
+				/>
+				<span v-else-if="slotProps.data.type === ParamType.EXPRESSION">Expression</span>
+				<Dropdown
+					v-else
+					:model-value="slotProps.data.type"
+					:options="typeOptions"
+					optionLabel="label"
+					optionValue="value"
+					placeholder="Select a parameter type"
+					@update:model-value="(val) => changeType(slotProps.data.value, val)"
+				/>
+			</template>
+		</Column>
+		<Column field="value" header="Initial value" style="width: 15%">
+			<template #body="slotProps">
+				<span v-if="slotProps.data.type === ParamType.MATRIX">Matrix</span>
+				<span v-else-if="slotProps.data.type === ParamType.EXPRESSION">
+					<InputText
+						class="p-inputtext-sm"
+						v-model.lazy="slotProps.data.value.expression"
+						@update:model-value="updateExpression(slotProps.data.value)"
+					/>
+				</span>
+				<div v-else-if="slotProps.data.type === ParamType.DISTRIBUTION">
+					<label>Min</label>
+					<InputNumber
+						class="p-inputtext-sm"
+						inputId="numericInput"
+						mode="decimal"
+						:min-fraction-digits="1"
+						:max-fraction-digits="10"
+						v-model.lazy="slotProps.data.value.distribution.parameters.minimum"
+						@update:model-value="emit('update-value', [slotProps.data.value])"
+					/>
+					<label>Max</label>
+					<InputNumber
+						class="p-inputtext-sm"
+						inputId="numericInput"
+						mode="decimal"
+						:min-fraction-digits="1"
+						:max-fraction-digits="10"
+						v-model.lazy="slotProps.data.value.distribution.parameters.maximum"
+						@update:model-value="emit('update-value', [slotProps.data.value])"
+					/>
```

Comment:
What resolution are you testing on, the min/max inputs are a bit messed up for me

<img width="897" alt="image" src="https://github.com/DARPA-ASKEM/terarium/assets/1220927/498d5b85-8b2a-4d2c-a2bc-c93082701fd4">

Reply:
<img width="589" alt="Screenshot 2024-01-23 at 11 20 18 AM" src="https://github.com/DARPA-ASKEM/terarium/assets/33158416/b3ced6b3-6c05-4e57-b574-db3cd3377f73">

I've added a
<img width="1651" alt="Screenshot 2024-01-23 at 11 21 35 AM" src="https://github.com/DARPA-ASKEM/terarium/assets/33158416/670626c6-d9d6-47fd-abba-cfbdac410cc1">
 class to the container so that the elements in the distribution will always show vertically: